<internal_error>
<sub_system>ERR</sub_system>
<callstack>
	0x2b3636266171: ccl_err + 0x8171 (_ZN15ERR_STACKWALKER15get_stack_traceEPPKviiPv + 0xb1)
	0x2b3636269015: ccl_err + 0xb015 (_Z14err_terminatorv + 0x4b)
	0x2b363e86f796: stdc++.so.6 + 0x63796 (_ZN10__cxxabiv111__terminateEPFvvE + 0x6)
	0x2b363e86f7c3: stdc++.so.6 + 0x637c3
	0x2b363e86fa36: stdc++.so.6 + 0x63a36 (__cxa_rethrow + 0x46)
	0x2b36128c6b64: QtCore.so.4 + 0x1a5b64 (_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x2f4)
	0x2b36128cb7c4: QtCore.so.4 + 0x1aa7c4 (_ZN16QCoreApplication4execEv + 0xb4)
	   0x401ea8: quartus + 0x1ea8 (_Z8qgq_mainiPPKc + 0x68)
	0x2b36339ac720: ccl_msg + 0x40720 (_Z15msg_main_threadPv + 0x10)
	0x2b363669d02c: ccl_thr + 0x602c (thr_final_wrapper + 0xc)
	0x2b36339ac7df: ccl_msg + 0x407df (_Z18msg_thread_wrapperPFPvS_ES_ + 0x62)
	0x2b3636047559: ccl_mem + 0xa559 (_Z18mem_thread_wrapperPFPvS_ES_ + 0x99)
	0x2b3636266f92: ccl_err + 0x8f92 (_Z18err_thread_wrapperPFPvS_ES_ + 0x27)
	0x2b363669d3f2: ccl_thr + 0x63f2 (thr_thread_wrapper + 0x15)
	0x2b36339ae7e2: ccl_msg + 0x427e2 (_Z12msg_exe_mainiPPKcPFiiS1_E + 0xa3)
	   0x401f7a: quartus + 0x1f7a (main + 0x26)
	0x2b363f2556e5: c.so.6 + 0x206e5 (__libc_start_main + 0xf5)
	   0x401d59: quartus + 0x1d59
</callstack>
<error>*** Fatal Error: Unhandled Exception</error>
<date>Fri Nov 10 21:57:19 2017
</date>
<version>Version 17.0.2 Build 602 07/19/2017 SJ Lite Edition</version>
</internal_error>